php/classes/renderers/class-renderer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace SeriouslySimplePodcasting\Renderers;
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class Renderer {
+
+	public function render($data, $template_path){
+		// enqueue any css or js, based ont he data being passed
```

Comment:
Yes! As commented earlier, let's move enqueues to the controller for now, and once we have that working, we can look at how we can do it on the renderer, based on the data.